app/services/course_exports/prepare_user_standings_export_service.rb

```diff
@@ -1,44 +1,50 @@
 module CourseExports
   class PrepareUserStandingsExportService
+    HEADER_ROW = [
+      "User ID",
+      "Email address",
+      "Name",
+      "Standing",
+      "Log entry",
+      "Created at",
+      "Created by",
+      "Archived at",
+      "Archived by"
+    ].freeze
+
+    BATCH_SIZE = 1000.0.freeze
```

Comment:
The `freeze` doesn't do anything useful here. The assignment of `1000.0` to `BATCH_SIZE` is a _pass-by-value_, not a _pass-by-reference_.

It's useful for the array above because it prevents the array from being modified (the assignment to `HEADER_ROW` is a _pass-by-reference_).

```rb
BLAH = ["foo", "bar"].freeze
BLAH2 = ["foo", "bar"]

BLAH << "frozen"
# can't modify frozen Array: ["foo", "bar"] (FrozenError)

BLAH2 << "unfrozen"
BLAH2
# => ["foo", "bar", "unfrozen"]
```